### PR TITLE
Use Doctum in replacement of Sami

### LINKED ISSUE #27: Documentation 404
The Documentation URL: https://develdocs.phpmyadmin.net/sql-parser/ is 404 error

### PR DESCRIPTION
**Pull request by williamdes**
Closes #27 
Ref: https://github.com/code-lts/doctum/issues/2

### CODE REVIEW

File: develdocs/doctum.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+$SOURCE = 'src';
+
+if (is_dir($_CLI['root'] . '/src') === false) {
+    $SOURCE = 'libraries';
+}
+
+$_CLI['root'] = rtrim($_CLI['root'], '/') . '/';
+
+$output = '<?php
```
Comment (mauriciofauth) [inline develdocs/doctum.php:13]: I think it's better to use Nowdoc string with sprintf for the variables.

Comment (williamdes) [reply develdocs/doctum.php]: Good idea, done :)

**Comment by williamdes** [comment]
@ibennetch I improved the index page of https://develdocs.phpmyadmin.net/ but the documentation does not seem to be building

**Comment by williamdes** [comment]
Maybe https://github.com/nikic/PHP-Parser/issues/687

**Pull request merged**